File: service/src/main/java/ru/diasoft/micro/dofacade/mock/CreateLoanReq.java
package ru.diasoft.micro.dofacade.mock;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreateLoanReq {

    @JsonProperty("NAME")
    String name;

    @JsonProperty("MOBILE_NUMBER")
    String mobileNumber;

    @JsonProperty("LEAD_SOURCE")
    String leadSource;

    @JsonProperty("IC_NUMBER")
    String idCardNumber;

    @JsonProperty("LEAD_NUMBER")
    String identityPercent  = "RH_132";

}

File: src/main/java/ru/diasoft/micro/dofacade/json/loan/CreateLoanReq.kt
package ru.diasoft.micro.dofacade.json.loan

import com.fasterxml.jackson.annotation.JsonProperty

data class CreateLoanReq(

    @JsonProperty("NAME")
    val name: String,

    @JsonProperty("MOBILE_NUMBER")
    val mobileNumber: String,

    @JsonProperty("LEAD_SOURCE")
    val leadSource: String,

    @JsonProperty("IC_NUMBER")
    val idCardNumber: String,

    @JsonProperty("LEAD_NUMBER")
    val identityPercent: String = "RH_132"
)

